_includes/release-notes/51known-issues.md

```diff
@@ -28,4 +28,29 @@ To workaround this issue, users can either:
 When a {{site.productname}} editor set to `inline: true` is opened on a mobile device, long toolbars will extend beyond the screen edge.
 
 #### Workaround
-Setting the editor [width]({{site.baseurl}}/configure/editor-appearance/#width) will constrain the toolbar width.
\ No newline at end of file
+Setting the editor [width]({{site.baseurl}}/configure/editor-appearance/#width) will constrain the toolbar width.
+
+### UI issue with sidebars on mobile
+
+#### Issue
+
+> **Note**: This issue affects the [Comments plugin]({{site.baseurl}}/plugins/comments/).
+
+When opening sidebars on small screens (such as mobile phones), the sidebar may occupy most or all of the text area of the editor.
+
+#### Workaround
+For the comments plugin, the comments sidebar can be collapsed using the `showcomments` toolbar button (![comment.svg]({{ site.baseurl }}/images/icons/comment.svg)).
+
+For other plugins with sidebars, no workaround exists. {{site.companyname}} recommends setting the `plugins` for `mobile` to exclude plugins with sidebars:
```

Comment:
Other plugins that use sidebars should be able to use the same workaround as comments, as when you register a sidebar TinyMCE will register a toolbar button with the same name. So the users would just need to make sure they include the toolbar button in their toolbar config.

Reply:
updated